openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -71,3 +71,36 @@ export function initEdit() {
     update_len();
     show_hide_title();
 }
+
+/**
+ * Initializes links element on edit page.
+ *
+ * Assumes presence of elements with id:
+ *    - '#links'
+ *    - '#link-label'
+ *    - '#link-url'
+ *    - '#link-errors'
+ */
+export function initEditLinks() {
+    $('#links').repeat({
+        vars: {
+            prefix: '$prefix'
```

Comment:
The variable $prefix originates in the Python template so must be communicated down to the code some how. i would suggest using data-prefix on the #links element and using `prefix: $('#links').data('prefix')` here.

Reply:
Thanks!